Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp

```diff
@@ -471,29 +471,24 @@ ErrorOr<NonnullRefPtr<TinyVGDecodedImageData>> TinyVGDecodedImageData::decode(St
     return TRY(adopt_nonnull_ref_or_enomem(new (nothrow) TinyVGDecodedImageData({ header.width, header.height }, move(draw_commands))));
 }
 
-void TinyVGDecodedImageData::draw_transformed(Painter& painter, AffineTransform transform) const
+void TinyVGDecodedImageData::draw(Painter& painter) const
 {
-    // FIXME: Correctly handle non-uniform scales.
-    auto scale = max(transform.x_scale(), transform.y_scale());
     for (auto const& command : draw_commands()) {
-        auto draw_path = command.path.copy_transformed(transform);
+        auto draw_path = command.path;
         if (command.fill.has_value()) {
-            auto fill_path = draw_path;
-            fill_path.close_all_subpaths();
+            draw_path.close_all_subpaths();
             command.fill->visit(
```

Comment:
This is subtly different behaviour than before: Previously, the closed path would be used for the fill, but the potentially-unclosed path for the stroke. Now you use the closed path for both.

Reply:
You're right let me fix that

Reply:
Done